### DISCUSSION TITLE
Questions about Theming & Fonts

### DISCUSSION THREAD

**Discussion by OfflinePing** [AllenDang/giu#839] [Q&A] [answered]
Hey,
Is there a quick way to change the style of every button, label and globally set the Font to a different. Also how would I customize a Window / Single Window without CSS like background color

**Comment by gucio321** [comment, marked as answer]
For fonts you can use font atlas
For style you can either use CSS main tag, use imgui push/pop style or use style setter

**Reply by OfflinePing** [reply]
Is there an example for the Font Atlas?

**Reply by gucio321** [reply]
Yeah iirc multiplefonts or something like that
